File: PROGRAMMERS/lv0/피자 나눠 먹기 (2).js
function solution(n) {
  let GCD = (a, b) => (a % b == 0 ? b : GCD(b, a % b));
  return parseInt(n / GCD(n, 6));
}

let inps = [6, 10, 4];

for (let i of inps) {
  console.log(solution(i));
}

File: PROGRAMMERS/lv0/python/피자 나눠 먹기 (2).py
def solution(n):
    GCD = lambda a,b : b if a%b==0 else GCD(b, a%b)
    return n // GCD(n, 6)

inps = [
    6,
    10,
    4
]

for inp in inps:
    print(solution(inp))